news-bundle/src/Controller/Page/NewsFeedController.php

```diff
@@ -54,8 +54,8 @@ public function __invoke(Request $request, PageModel $pageModel): Response
         $baseUrl = $staticUrl ?: $request->getSchemeAndHttpHost();
 
         $feed = (new Feed())
-            ->setTitle($pageModel->title)
-            ->setDescription($pageModel->feedDescription)
+            ->setTitle(html_entity_decode($pageModel->title))
```

Comment:
why do we pass two arguments to `html_entity_decode` in `NewsFeedListener` but not here?

Reply:
Good catch, will fix this!

Reply:
Fixed in ddbbd62cf7